bot/code_review_bot/report/mail_builderrors.py

```diff
@@ -71,6 +71,10 @@ def publish(self, issues, revision):
             logger.info("Unable to find the author for commit.")
             return
 
+        logger.info(
+            "Sending build_error email to {}.".format(commit["author"]["email"])
+        )
```

Comment:
Please use the [structlog features](http://www.structlog.org/en/stable/getting-started.html):

```
logger.info("Send build error email", to=commit["author"]["email"])
```